features/support/env.js

```diff
@@ -42,44 +42,33 @@ module.exports = function () {
         this.OSRM_PORT = process.env.OSRM_PORT && parseInt(process.env.OSRM_PORT) || 5000;
         this.HOST = 'http://127.0.0.1:' + this.OSRM_PORT;
 
-        // TODO make sure this works on win
-        if (process.platform.match(/indows.*/)) {
+        if (process.platform.match(/^win.*/)) {
```

Comment:
`win32` is a platform name https://nodejs.org/api/process.html#process_process_platform